Global.h

```diff
@@ -721,6 +721,8 @@ namespace vstd
 		return false;
 	}
 
+	std::vector<std::string> split(std::string s, std::string separators);

```

Comment:
Add another .h to `include/vstd` dir. Put .cpp to `lib/vstd`

Reply:
And please do not include yet another helper to `Global.h`. I should only contain global compiler configuration and heavy 3rd party includes for PCH.